src/ArticleCard.jsx

```diff
@@ -0,0 +1,15 @@
+export const ArticleCard = ({ article }) => {
+	return (
+		<li className='card'>
+			<h2>{article.title}</h2>
+			<img
+				className='item-image'
+				src={article.article_img_url}
+				alt={`Image of ${article.title}`}
+			></img>
+			<p>Author: {article.author}</p>
+			<p>Comments: {article.comment_count}</p>
+			<p>Votes: {article.votes}</p>
+		</li>
+	);
+};
```

Comment:
This is the article card that displays on the home page...

When I click on one of them, what should happen?